### DISCUSSION TITLE
Not able to reproduce test error locally (fullstack-acct-management-level-1)

### DISCUSSION THREAD

**Discussion by DariBerrie** [DevSkillsHQ/discusssions#14] [Help] [answered]
Hello! My tests are passing locally, but I am unable to reproduce this error which happens in the pipeline but not when running tests:
```
Completed 500 Internal Server Error in 1038ms (ActiveRecord: 1.5ms | Allocations: 15999)
ActiveRecord::RecordNotUnique (SQLite3::ConstraintException: UNIQUE constraint failed: accounts.id):
```
I have gone through the tests step-by-step to understand why this may occur, but since it's not a problem using the same data locally, I'm not sure why SQLite has a problem in the pipeline. It might be causing my last test to not pass, but I'm not sure.

Cypress doesn't pass the last test for a transaction it is looking for, but I can see that the transaction was created and should have been checked. So I just don't know why these checks are happening locally but not in the pipeline.
`AssertionError: Timed out retrying after 4000ms: Expected to find element: `[data-type=transaction][data-account-id=fc25d54a-7d76-43d9-ac66-bf551d041f62][data-amount=30][data-balance=30]`, but never found it.`

**Comment by fenske** [comment, marked as answer]
Hi @DariBerrie. 👋 As you can see in the video below, the frontend never gets rendered so that's why the tests fail. I reproduced it locally and got the same behavior.

https://user-images.githubusercontent.com/1162212/228161327-c76941c0-5c9f-47e5-9f3a-d1f3d697a2ed.mp4
